pytorch/inference/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -165,8 +164,10 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
     # Below 2 are included in miniconda base, but not mamba so need to install
     conda-content-trust \
     charset-normalizer \
- && /opt/conda/bin/conda install -c pytorch magma-cuda115=2.6.1 \
- && /opt/conda/bin/conda clean -ya
+ && /opt/conda/bin/conda install -c pytorch magma-cuda115=2.6.1 
+# Moving after installs as this attempts to install conda 4.13 which is not compatible with mamba==0.22.1 and causes the build to fail.
+RUN /opt/conda/bin/conda update -y conda \
+   && /opt/conda/bin/conda clean -ya
```

Comment:
Does this step still install conda 4.13, then?

Reply:
Just like PR-1991 This will give a warning that 4.12 is installed and provides a command to update it manually. it does not do it automatically. If this is before the conda-forge installs, it will install automatically. Thus, it was moved to after the installed. This was based off the same usage in PR-1991

Reply:
Thanks for the explanation! Could you please amend the comment above the RUN statement, explaining the solution, rather than the problem it solves? Right now, the comment makes it seem as though the `conda update` step attempts to install 4.13 anyway, so we're just moving it to the end so that we can install packages **before** conda is updated to 4.13 in the image.